Utilities/File.cpp

```diff
@@ -966,7 +966,7 @@ fs::file::file(const std::string& path, bs_t<open_mode> mode)
 			info.is_writable = (basic_info.FileAttributes & FILE_ATTRIBUTE_READONLY) == 0;
 			info.size = this->size();
 			info.atime = to_time(basic_info.LastAccessTime);
-			info.mtime = to_time(basic_info.ChangeTime);
+			info.mtime = to_time(basic_info.LastWriteTime);
 			info.ctime = info.mtime;
```

Comment:
Could you change this line so that it keeps using ChangeTime? The mtime change looks to be correct but ctime should still be ChangeTime in my opinion.